thirdeye/thirdeye-frontend/app/pods/manage/alert/edit/route.js

```diff
@@ -97,15 +115,47 @@ export default Route.extend({
       granularity: `${bucketSize}_${bucketUnit}`
     });
 
-    controller.initialize();
+    // Populate 'alerts monitored' table with currently selected config group
+    get(this, 'prepareAlertList').perform(selectedConfigGroup);
   },
 
+  /**
+   * Fetch alert data for each function id that the currently selected group watches
+   * @method fetchAlertDataById
+   * @param {Object} functionIds - alert ids included in the currently selected group
+   * @return {RSVP.hash} A new list of functions (alerts)
+   */
+  fetchAlertDataById: task(function * (functionIds) {
+    const functionArray =  yield functionIds.map(id => fetch(selfServeApiCommon.alertById(id)).then(checkStatus));
```

Comment:
i wonder if this is yielding correctly. 

vs like 
 ```const functionArray = functionIds.map(id => yield fetch(selfServeApiCommon.alertById(id)).then(checkStatus));```

Reply:
Yes, yielding fine (tested). Otherwise I get some error like "Unexpected strict mode reserved word"